src/constants.ts

```diff
@@ -69,6 +69,14 @@ export type SupportedRegistries = 'azurecr.io' | 'docker.io';
 
 export const browseItem: IAzureQuickPickItem<undefined> = { label: `$(file-directory) ${localize('browse', 'Browse...')}`, description: '', data: undefined };
 export const loadMoreQp: IAzureQuickPickItem = { label: '$(sync) Load More', data: undefined, suppressPersistence: true };
+
+export const noMatchingResources = 'noMatchingResources';
+export const noMatchingResourcesQp: IAzureQuickPickItem<typeof noMatchingResources> = {
```

Comment:
Makes me feel like this behavior should be implemented in the `ui.showQuickPick` command instead. If it has 0 picks, we could just show this.

Reply:
Yeah I agree, we should probably revisit implementing this as part of the wrapper in tools later